src/coreclr/nativeaot/System.Private.CoreLib/src/Internal/Runtime/CompilerServices/OpenMethodResolver.cs

```diff
@@ -52,6 +55,7 @@ public unsafe OpenMethodResolver(RuntimeTypeHandle declaringTypeOfSlot, RuntimeM
             _handle = handle;
             _readerGCHandle = readerGCHandle;
             _nonVirtualOpenInvokeCodePointer = IntPtr.Zero;
+            s_lazyGvmLookupForSlot = &TypeLoaderExports.GVMLookupForSlot;
```

Comment:
```suggestion
            if (s_lazyGvmLookupForSlot == null)
                s_lazyGvmLookupForSlot = &TypeLoaderExports.GVMLookupForSlot;
```
Avoid cache trashing?